### ISSUE TITLE
[Feature Request] Heuristics for merging metadata

### ISSUE THREAD

**Issue by hseg** [papis/papis#705]
Especially when a document is matched multiple times (eg on arxiv + a couple of journals), it would make sense to have papis notice one is an updated version of the other and recommend eg the journal article's metadata over the preprint's.
Also, in cases where there are more than two sets of metadata, it is sometimes difficult to get a clear picture of what the maximal consistent set of metadata is (eg, have had cases where one extractor had affiliation info but the other had the journal info)

**Comment by alejandrogallo** [comment]
Agreed, but this is quite difficult, it also affects the UI, it is quite a mess but we/I have not been able to figure out a sensible algorithm and workflow for this that one should then go ahead and implement, that could be though a nice project
